File: palin.rb
class Palin
	def check_palindrome
		puts("Enter Number:")
		n=gets.to_i
		num=n
		rev=0
		while(n!=0) do
			rev=(rev*10)+(n%10)
			n/=10
			end
		if rev==num then puts("Palindrome") else puts("Not Palindrome") end
		end
end

ob=Palin.new
ob.check_palindrome

File: javascript_assignment/palin.js
function palindrome(){

var n = 1321;
var num=n;
var rev;
while(num!==0){
	rev=(rev*10)+(num%10);
	num/=10;
}

if(rev===n)
console.log(true);
else
console.log(false);
}
console.log(palindrome())
